data/schema.yaml

```diff
@@ -228,6 +228,18 @@ components:
 
     # schema definitions for referred blocks below
 
+    textAttribute:
+      enum:
+        - italic
+        - bold
+      example: italic
+      type: string
+
+    textAttributes:
```

Comment:
A comment to clarify the use case for this in Apps would be good as currently this appears to be orphaned

Reply:
👍